config/environments/test.rb

```diff
@@ -39,4 +39,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Devise config
+  config.action_mailer.default_url_options = { host: ENV.fetch('LOCALHOST') }
```

Comment:
This will be missing the post (it will be just `localhost` not `localhost:3000`, you could use `Rails.configuration.admin_host` or set the port directly here.

Reply:
Actually it will be `lvh.me`. See the override in `.env.test`.

Reply:
Ah right, of course